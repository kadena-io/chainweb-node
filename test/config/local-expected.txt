{"status":"success","data":{"amount":"-1.00","data":{"transfer-to":"Acct2"},"balance":"999999.00","account":"Acct1"}}