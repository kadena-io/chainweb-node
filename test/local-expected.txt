test-group: local
results:
  gas: 0
  result:
    status: success
    data:
      amount: -1
      data:
        transfer-to: Acct2
      balance: 999999
      account: Acct1
  reqKey: sJcgrbkD-jEXnZth6mpDIcMi8E3dGr8CKw9g849FnDc
  logs: wsATyGqckuIvlm89hhd2j4t6RMkCrcwJe_oeCYr7Th8
  metaData: null
  continuation: null
  txId: null
