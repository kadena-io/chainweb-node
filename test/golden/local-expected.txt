test-group: local
results:
  status: success
  data:
    obj:
      amount:
        i: null
        lit: -1
      data:
        obj:
          transfer-to:
            i: null
            lit: Acct2
        type: '*'
        i: null
      balance:
        i: null
        lit: 999999
      account:
        i: null
        lit: Acct1
    type: '*'
    i: null
